maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/SurefireHelper.java

```diff
@@ -286,7 +289,15 @@ private static String createErrorMessage( SurefireReportParameters reportParamet
         }
         else
         {
-            msg.append( "There are test failures.\n\nPlease refer to " )
+            if ( result.getFailures() > 0 )
```

Comment:
@oehme 
Pls see the class `IntegrationTestPlugin`. It reports this text other way. The same messages should be there as well but the solution is different from this one.

Reply:
The IntegrationTestMojo does not report this text. The VerifyMojo does and uses the same logic as the SurefirePlugin

Reply:
@oehme 
ok just fine!